### PR TITLE
Closes #16 - add two new RPC nodes

### LINKED ISSUE #16: Add two new RPC nodes to public nodes list
There have been two new RPC nodes added by members of the community:

@steemgigs added: steemd.steemgigs.org
@ausbitbank added: rpc.steemviz.com

### PR DESCRIPTION
**Pull request by TimCliff**

### CODE REVIEW

**Comment by steemgigs** [comment]
Thank you very much @TimCliff

File: _quickstart/steemd_nodes.md [hunk 1]
```diff
@@ -28,8 +28,10 @@ Applications that interface directly with the Steem blockchain will need to conn
 |gtg.steem.house:8090|@gtg|
 |steemd.minnowsupportproject.org|@followbtcnews|
 |steemd.privex.io|@privex|
+|steemd.steemgigs.org|@steemgigs|
 |steemd.steemit.com|@steemit|
 |rpc.steemliberator.com|@netuoso|
+|rpc.steemviz.com|@ausbitbank|
```
Comment (plink01001) [inline _quickstart/steemd_nodes.md]: The `rpc.steemviz.com` server is currently returning a 502 error. Is this still supported?

**Comment by TimCliff** [comment]
@plink01001 yes, the server is still operational. I checked with AusBitBank who maintains the server, and he said it is just doing a reply after a maintenance update.

**Comment by plink01001** [comment]
Passed testing and code review. @relativityboy to approve/merge.

**Comment by TimCliff** [comment]
@relativityboy can this be merged, so the site can have an updated list of RPC nodes?

**Pull request merged**